templates/webpack-default/app/views/profile/index.xml

```diff
@@ -0,0 +1,31 @@
+<Alloy>
+	<Tab title="Profile">
+		<Window title="Profile">
+			<ScrollView id="scrollview">
+				<ImageView id="profile" />
+				<Label class="profileName">Verona Blair</Label>
+				<TableView onClick="doClick">
+					<TableViewSection id="settingsSection">
+						<HeaderView>
+							<View class="separatorWrapper">
+								<View class="separator" />
+							</View>
+						</HeaderView>
+						<Require src="/profile/row" iconBG="#eef0f6" icon="user" iconColor="#a9cdd1" label="Personal Details" labelColor="#000" />
```

Comment:
I know we don't have any style rules for this, but the prettier setting for Vue for example tends to wrap attributes to new lines ones the line gets over a certain amount of character. I find that easier to read than a single line that goes on and on.

```suggestion
						<Require
							src="/profile/row"
							iconBG="#eef0f6"
							icon="user"
							iconColor="#a9cdd1"
							label="Personal Details"
							labelColor="#000"
						/>
```

Reply:
Updated the Require tags to look like that